components/raftstore/src/store/peer.rs

```diff
@@ -277,19 +277,19 @@ pub struct CheckTickResult {
     reason: &'static str,
 }
 
-pub struct ProposedAdminCmd<S: Snapshot> {
+pub struct ProposedAdminCmd<C: ErrorCallback> {
```

Comment:
Why need to add `ErrorCallback`?

Reply:
Same logic as you did for ReadIndexQueue, ReadIndexRequest etc. 
CmdEpochChecker and ProposedAdminCmd can be reused in v2.

Reply:
I added it because they were used. It doesn't seem to be used in `ProposeAdminCmd`.